packages/rsk3-account/tests/models/account.test.js

```diff
@@ -496,13 +420,9 @@ describe('AccountTest', () => {
 
         expect(() => {
             Account.fromPrivateKey(mockKey).toV3Keystore('password', {kdf: 'nope'});
-        }).toThrow('Unsupported kdf');
+        }).toThrow('options.n should be number and has value of 2048, 4096, 8192 or 16384');
 
         expect(fromPrivate).toHaveBeenCalledWith(mockKey);
-
-        expect(randomBytes).toHaveBeenNthCalledWith(1, 32);
-
-        expect(randomBytes).toHaveBeenNthCalledWith(2, 16);
```

Comment:
This is a failed test case, randomBytes() is not called so the rest of toHaveBeenNthCalledWith() checks are not needed.